docs/theory/force_calibration/fitting.rst

```diff
@@ -37,3 +37,20 @@ Setting the number of points per block too low would result in a bias from insuf
 (:math:`R_f`) and an underestimation of the distance response (:math:`R_d`). In practice, one should
 use a high number of points per block (:math:`n_b \gg 100`), unless a very low corner frequency precludes this.
 In such cases, it is preferable to increase the measurement time.
+
+Goodness of fit
+---------------
+
+Based on the model and noise assumptions, we can calculate a goodness of fit criterion.
+When sufficient blocking has taken place, the sum of squared residuals that is being minimized during the fitting procedure is distributed according to a chi-squared distribution characterized by :math:`N_{dof} = N_{data} - N_{free}` degrees of freedom.
+Here :math:`N_{data}` corresponds to the number of data points we fitted (after blocking) and :math:`N_{free}` corresponds to the number of parameters we fitted.
+We can use the value we obtain to determine how unusual the fit error we obtained is.
+
+.. math::
+
+    \mathrm{support} = 100 P(x > \chi_{obtained}^2) = 100 \int_{\chi_{obtained}^2}^{\infty} \chi^2_{N_{dof}}(x) dx
+
+The support or backing is the probability that a repetition of the measurement that produced the data we fitted to, will produce data that when fitted results in a larger value than the one we obtained.
```

Comment:
No idea if this still relays the message properly, but I hoped to be a bit more clear.

```suggestion
The support or backing is the probability that a repetition of the measurement that produced the data we fitted to, will produce, after fitting, residuals of which the squared sum is larger than the one we obtained initially.
```

Reply:
Much better. Thank you 🙌